src/app/modules/landing/modules/auth/components/login/login.component.ts

```diff
@@ -40,7 +41,13 @@ export class LoginComponent {
           const { returnUrl } = this.route.snapshot.params;
           this.store.dispatch(new Navigate([returnUrl || '/']));
         },
-        (err: HttpErrorResponse) => this.store.dispatch(new ToasterError(extractError(err))),
+        (err: HttpErrorResponse) => {
```

Comment:
new AuthLogin({ ...}).subscribe({
   error: err => {....}
})

Reply:
it is not possible to subscribe on AuthLogin directly